.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: python
```

Comment:
Thanks for doing the legwork to get Travis running. Any time I've set it up in the past, it's taken a *lot* of trial an error.